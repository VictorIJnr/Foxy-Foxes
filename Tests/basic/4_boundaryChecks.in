6
3
7
3
6
2
5
1
1
3
1
4
