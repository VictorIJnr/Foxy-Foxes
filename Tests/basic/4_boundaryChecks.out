     o o o    
     o o o    
 o o o o o o o
 o o . . . o o
 . . . . . . .
     . * .    
     . . .    
Geese play. Enter move:
Illegal move!
     o o o    
     o o o    
 o o o o o o o
 o o . . . o o
 . . . . . . .
     . * .    
     . . .    
Geese play. Enter move:
Illegal move!
     o o o    
     o o o    
 o o o o o o o
 o o . . . o o
 . . . . . . .
     . * .    
     . . .    
Geese play. Enter move:
     o o o    
     o o o    
 o o o o o o o
 o . . . . o o
 . o . . . . .
     . * .    
     . . .    
Fox plays. Enter move:
End of input reached. Goodbye.
